### DISCUSSION TITLE
Extract_Text extract character of embedded PDF

### DISCUSSION THREAD

**Discussion by flashpixx** [jsvine/pdfplumber#981] [Ask for help with specific PDFs]
Hello,

I have got a LaTeX generated PDF, which uses the `\includepdf` call to embed other PDF. Now I'm using PDFPlumber to extract the text (it is a test-case for any PDF later). In general extract_text works fine, but on the embedded pages I get "character garbage" back e.g. (partly extracted)

```
 oo(cid:212)iiioiIIIII
               2nov2
       .glofrE   
   dnu           
     eleiv       
    .trh(cid:159)feghcrud suarK
 erawtfos III -snoitargetnI reb(cid:159)negeg
     tiebrA      
      red      (cid:209)
   ssezorpgnitseT dnu leiv
  iiil sesaC-tseT ruz nrreH 99-23 ed.erawtfos-vdp.~
      tieZ       
     red         
     tztesegmu nihretiew
     renies     hcsiweiN
   neigolonhceT  
 1 II           75)12350(
     gnuressebreV ezruk
      ehcsn(cid:159)w
    gidn(cid:138)tsnegie netlahreV
```

How can I avoid that this text is returned on extract_text for a whole page, which contains another pdf? Is it possible this is a font information?

Thanks

**Comment by jsvine** [comment]
Hi @flashpixx, and very interesting. I'm not familiar with LaTeX's `\includepdf` feature. Would you be able to attach an example PDF that reproduces this problem?
